src/i18n/ru/index.js

```diff
@@ -97,59 +97,59 @@ export default {
           label: 'Quantum'
         },
         KeyboardSettings: {
-          label: 'Keyboard Settings'
+          label: 'Настройки клавиатуры'
         },
         AppMediaMouse: {
-          label: 'App, Media and Mouse'
+          label: 'Приложение, медиа и мышь'
         }
       },
       settingsPanel: {
-        title: 'Configurator Settings',
+        title: 'Настройки конфигуратора',
         fastInput: {
           label: 'Fast Input',
           title: 'ctrl + alt + f',
-          help: 'Input keys via keyboard without clicking each position.'
+          help: 'Ввод клавиш через клавиатуру без нажатия на каждую позицию.'
         },
         displaySizes: {
-          label: 'Show Key Sizes',
+          label: 'Показать размеры клавиш',
           title: 'ctrl + alt + u',
-          help: 'Show keycap sizes in Key Units'
+          help: 'Показать размеры клавиш в Key Units'
         },
         toggleTutorial: {
-          label: 'Video Tutorial',
-          title: 'MechMerlin how-to video on Configurator',
-          help: 'MechMerlin video guide'
+          label: 'Видеоурок',
+          title: 'MechMerlin видео с инструкциями о конфигураторе',
+          help: 'MechMerlin видеогид'
         },
         darkmode: {
-          label: 'Toggle Darkmode',
-          title: 'Dark mode'
+          label: 'Переключить в темный режим',
+          title: 'Темный режим'
         },
         language: {
-          title: 'Language'
+          title: 'Язык'
         },
         on: {
-          label: 'On'
+          label: 'Включить'
         },
         off: {
-          label: 'Off'
+          label: 'Выключить'
         }
       },
       errors: {
         invalidQMKKeymap:
-          "Sorry, that doesn't appear to be a valid QMK keymap file.",
+          "Извините, это неверный файл раскладки QMK.",
         kbfirmwareJSONUnsupported:
-          "Sorry, QMK Configurator doesn't support importing kbfirmware JSON files.",
-        unknownJSON: "Sorry, this doesn't appear to be a QMK keymap file.",
-        unsupportedBrowser: "You're using a non supported browser. Please use"
+          "Извините, QMK конфигуратор не поддерживает импорт JSON файлов раскладки прошивки.",
+        unknownJSON: "Извините, это не похоже на файл раскладки QMK.",
+        unsupportedBrowser: "Вы используете не поддерживаемый браузер. Пожалуйста, используйте"
       },
       statsTemplate:
-        '\nLoaded {layers} layers and {count} keycodes. Defined {any} Any key keycodes\n',
+        '\nLoaded {layers} слои и {count} коды клавиш. Определен {any} любой код кодов клавиш\n',
       maintain:
-        'This project is maintained by QMK collaborators and contributors like you!',
-      hostedOn: 'Hosted on GitHub Pages',
-      serverStatus: 'Server Status',
-      apiVersion: 'API Version',
-      jobsWaiting: 'job(s) waiting'
+        '"Данный проект поддерживается коллабораторами QMK и такими же соавторами, как ты!"',
```

Comment:
I guess this is the reason of Travis failure.

Reply:
Yes, thank you, my fault